gap/projective/c3c5.gi

```diff
@@ -178,7 +178,8 @@ function(ri, G)
   # There might be a kernel, because we have more scalars over the bigger
   # field, so go for it, however, fewer generators should suffice:
   # Also, doing normal closure will not help!
-  findgensNmeth(ri).args := [5,0];
+  findgensNmeth(ri).method := FindKernelRandom;
+  findgensNmeth(ri).args := [5];
```

Comment:
I only know in theory what this is supposed to do, but I don't know enough about the package to understand what this change does. Is "FindKernelRandom" the default method and you're clarifying that here?

Reply:
It's not the default method. The default method additionally does normal closure computations, that is it also takes conjugates of the found kernel generators. Here we don't do conjugations, though. Previously this was done by passing a 0 as second argument to the default method. Now, I simply set the method which does not do kernel computations.

Reply:
Ah, thanks for explaining!